.devcontainer/Dockerfile

```diff
@@ -310,14 +312,47 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     make -j"$(nproc)" && \
     make install && \
     cd / && \
-    rm -rf freediameter
+    rm -rf freediameter \
+    rm -rf /tmp/*
 
 ##### Go language server support for vscode
 RUN go get -v golang.org/x/tools/gopls
 
 #### Update shared library configuration
 RUN ldconfig -v
 
+
+##### Install Python requirements
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+
+### install python3-aioeventlet from the magma apt repo (from .circleci/config.yml)
+COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub ${MAGMA_ROOT}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub
+RUN cat ${MAGMA_ROOT}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub | sudo apt-key add - \
+    && echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" | sudo tee /etc/apt/sources.list.d/fbc.list \
+    && sudo apt-get update -y  \
+    && sudo apt-get install -y python3-aioeventlet \
+    && sudo rm -rf /var/lib/apt/lists/*  # delete lists that are downloaded by apt-get update
+
+ARG PATCH_DIR=lte/gateway/deploy/roles/magma/files/patches
```

Comment:
As this is only used once, we could just inline it. Same with DIST_PACKAGES_DIR.

Reply:
Yes, true. (I already tried to include the remaining patches and didn't inline the variables again...)